bundle/manifests/gitops-operator-controller-manager-metrics-service_v1_service.yaml

```diff
@@ -3,14 +3,14 @@ kind: Service
 metadata:
   creationTimestamp: null
   labels:
-    control-plane: controller-manager
+    name: openshift-gitops-operator
```

Comment:
What is the reason to change these labels?

Reply:
These labels are not unique and pod labels should be unique, For example a user raised an issue with us that GitLab operator also use the same labels which is causing the GitLab operator service to forward the requests to GitOps operator pod as the labels are same.

Reply:
Which label needs to be unique? Do all labels need to be unqiue? I see you added the `name` label and removed the `control-plane` label. Any idea what is the `control-plane` label is for?

Reply:
Just wanted to make sure that it is safe to remove the `control-plane` label.

Reply:
If I understand the issue correctly, it seems to me a safer fix is to make the `control-plane` label` unique instead of removing it. Not sure why we need a  new `name` label.
```
control-plane: openshift-gitops-operator
```
Also, it should be fixed in upstream (with a different name).

Reply:
I tried to keep the label generic to as what other operators are using. We can surely keep the `key` as `control-plane` and just change the value to `openshift-gitops-operator` as well